evap/staff/importers.py

```diff
@@ -653,21 +680,29 @@ def process_contributors(self, evaluation, test_run, user_list):
         already_related = [contribution.contributor for contribution in already_related_contributions]
         if already_related:
             msg = format_html(_("The following {} users are already contributing to evaluation {}:"), len(already_related), evaluation.full_name)
+            msg = format_html(ngettext('The following user is already contributing to evaluation %(name)s:', 'The following %(users)d users are already contributing to evaluation %(name)s:', len(already_related),) % {'users': len(already_related),'name': evaluation.full_name})
```

Comment:
this should be `+=` or the line above should be removed

Reply:
We don't usually do percent-formatted strings, please use [f-strings](https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/) instead